www/printing/oracle/index.php

```diff
@@ -1,93 +0,0 @@
-<!DOCTYPE html>
```

Comment:
we haven't ported oracle to ocfweb yet, have we? deleting this seems premature.

Reply:
Isn't it at https://www.ocf.berkeley.edu/stats/printing/pages-printed?

Reply:
We already have the current stats pages that display the current amount of toner in the printer, and the data is still there. Do we need a historical chart right now?

Reply:
That only shows the pages printed, not the toner levels. @sahilhasan (or someone in his family) is replacing the toner graph, since it isn't useful at all as-is because of the scaling issues, so it'll be replaced with something else. Since this graph isn't useful right now anyway, it makes sense to just remove it and potentially replace it with something later if needed.

Reply:
Do we feel the need to actually delete this though?

Reply:
I mean we don't need to actually delete it, it could be left here and the python stuff extracted out and put into other repos I suppose?